classes/class-wc-connect-shipping-method.php

```diff
@@ -274,6 +275,7 @@ public function calculate_shipping( $package = array() ) {
 					),
 					__FUNCTION__
 				);
+				return;
```

Comment:
I opted to keep the second return statement since technically this function isn't supposed to return any value.